common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -276,21 +269,22 @@ status_function() {
     pgrep -l ieee1905_transport
     pgrep -l local_bus
 
-    # check for operational status
-    LOGS_PATH=@BEEROCKS_LOG_FILES_PATH@
+    bridge_mac="$(ifconfig @BEEROCKS_BRIDGE_IFACE@ | grep -o -E '([[:xdigit:]]{1,2}:){5}[[:xdigit:]]{1,2}')"
+    bml_cmd="bml_get_device_operational_radios $bridge_mac"
+    dbg "Executing BML: $bml_cmd"
 
     error=0
-    report "Main agent operational" main_agent_operational $LOGS_PATH
-    report "@BEEROCKS_WLAN1_IFACE@ radio agent operational" radio_agent_operational $LOGS_PATH @BEEROCKS_WLAN1_IFACE@
-    report "@BEEROCKS_WLAN2_IFACE@ radio agent operational" radio_agent_operational $LOGS_PATH @BEEROCKS_WLAN2_IFACE@
-
-    [ "$VERBOSE" = "true" ] && [  $error = 1 ] && {
-        cat $LOGS_PATH/beerocks_agent@BEEROCKS_LOG_FILES_SUFFIX@
-        cat $LOGS_PATH/beerocks_agent_@BEEROCKS_WLAN1_IFACE@@BEEROCKS_LOG_FILES_SUFFIX@
-        cat $LOGS_PATH/beerocks_agent_@BEEROCKS_WLAN2_IFACE@@BEEROCKS_LOG_FILES_SUFFIX@
-    }
-
-    exit $error
+    execute_bml "$bml_cmd"
+    dbg $output
+    OK_count=`echo "$output" | grep -c -e "OK.*operational"`
```

Comment:
You should probably add a comment describing the expected output and what exactly this conditions means